### PR TITLE
feat(replays): Use new error column for replays

### PR DESCRIPTION
**Pull request by JoshFerge**
- Use new error columns for replay error counts and filtering
- Old error ids will still be retrieved in final results, but not be used in counts or searchable. We filter them out if no corresponding error exists anyways on replay details, so effectively, this will fix the counts.
- Will make a follow-up PR to remove old error column entirely from final results.

### CODE REVIEW

File: tests/sentry/replays/test_organization_replay_index.py [hunk 1]
```diff
@@ -1968,10 +1973,8 @@ def test_event_id_count_columns(self):
         with self.feature(REPLAYS_FEATURES):
             response = self.client.get(
                 self.url
-                + f"?field=id&field=new_count_errors&field=count_warnings&field=count_infos&query=id:{replay1_id}"
+                + f"?field=id&&field=count_warnings&field=count_infos&query=id:{replay1_id}"
```
Comment (cmanallen) [inline tests/sentry/replays/test_organization_replay_index.py:1976]: Double ampersand.

**Review by cmanallen — APPROVED** [review]
> not be used in counts or searchable.

This only affects the replays index page, correct?  No other pages rely on the error_ids or count_errors parameters?

**Comment by JoshFerge** [comment]
> > not be used in counts or searchable.
> 
> This only affects the replays index page, correct? No other pages rely on the error_ids or count_errors parameters?

yup, just the index.

File: tests/sentry/replays/test_organization_replay_index.py (hunk 1 above)
Comment (JoshFerge) [reply tests/sentry/replays/test_organization_replay_index.py:1976]: ooo thank you!

**Pull request merged**